### ISSUE TITLE
Nullability operator in contracts implementation

### ISSUE THREAD

**Issue by valentasm1** [Dreamescaper/GenerateAspNetCoreClient#14]
Now i get
```
[Get("/some-api")]
Task<ListResponse> GetSome([AliasAs("page_number")] int? pagenumber = null, [AliasAs("date_types")] DateType[] dateTypes = null, [AliasAs("q")] string query = null);
```
It should add nullability operator ? on string and array too. It should work only on query params since they are optional.
It should be
```
[Get("/some-api")]
Task<ListResponse> GetSome([AliasAs("page_number")] int? pagenumber = null, [AliasAs("date_types")] DateType[]? dateTypes = null, [AliasAs("q")] string? query = null);
```

**Comment by Dreamescaper** [comment]
Yeah, it doesn't support NullableReferenceTypes as for now. I'm wondering though - what issues does is cause? I don't think it causes any warnings due to `<auto-generated />` tag, right?

**Comment by valentasm1** [comment]
Nothing serious. It is just compiler warnings.

**Comment by Dreamescaper** [comment]
But where?  Would have expected all compiler warnings to be disabled because of `<auto-generated />` tag...

**Comment by valentasm1** [comment]
I didnt knew it. Nice. It dont show warning in contract file. 
When i need to mock it for test, so it implements it without operator like this "string query = null".  Workaround would be add exceptions for compiler warning, add ? manually.
![image](https://user-images.githubusercontent.com/8047891/203299088-deefb9e7-fce9-424a-b01e-42afc0a5550e.png)
